.github/workflows/tests.yml

```diff
@@ -82,7 +82,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install -r requirements.pinned.txt
+          pip install -r requirements.txt
```

Comment:
We will need to update also this: https://gitlab.cern.ch/cds-team/cds-videos-openshift/-/blob/master/scripts/build.sh?ref_type=heads#L18